libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -838,6 +856,45 @@ void _IotMqtt_DecrementConnectionReferences( _mqttConnection_t * pMqttConnection
 
 /*-----------------------------------------------------------*/
 
+static uint32_t getTimeMs( void )
+{
+    uint32_t timeMs;
+
+    return ( uint32_t ) IotClock_GetTimeMs();
+}
+
+/*-----------------------------------------------------------*/
+
+static int32_t transportSend( NetworkContext_t networkContext,
+                              const void * pMessage,
+                              size_t bytesToSend )
+{
+    int32_t bytesSend = 0;
+
+    for( size_t i = 0; i < MAX_NO_OF_MQTT_CONNECTIONS; i++ )
+    {
+        if( connToContext[ i ].mqttConnection->pNetworkConnection == networkContext )
+        {
+            /* Sending the bytes on the network using Network Interface of MQTT v4_beta1 library. */
+            bytesSend = connToContext[ i ].mqttConnection->pNetworkInterface->send( connToContext[ i ].mqttConnection->pNetworkConnection, ( uint8_t * ) pMessage, bytesToSend );
+
+            if( bytesSend < 0 )
+            {
+                /*need to include iot_network_freertos.h */
```

Comment:
/*need to include iot_network_freertos.h */ -> Not sure I got this comment. What do you mean?